src/mplex.ts

```diff
@@ -14,9 +14,12 @@ import type { Components } from '@libp2p/interfaces/components'
 import type { Sink } from 'it-stream-types'
 import type { StreamMuxer, StreamMuxerInit } from '@libp2p/interfaces/stream-muxer'
 import type { Stream } from '@libp2p/interfaces/connection'
+import type { MplexInit } from './index.js'
 
 const log = logger('libp2p:mplex')
 
+const MAX_STREAMS_PER_CONNECTION = 1024
```

Comment:
newbie question: is the TypeScript way of doing things to define defaults here rather than where the override can be set?  For example, I was surprised not to see this closer to MplexInit.

Reply:
`MplexInit` is defined in `index.ts`, exporting it for import into `mplex.ts` would also expose it to people importing `@libp2p/mplex`.  It could be the other way round though, defined in `mplex.ts` and imported into `index.ts`.